cloud/aws/templates/aws_oidc/variables.tf

```diff
@@ -472,3 +472,9 @@ variable "monitoring_stack_enabled" {
   description = "If true, Prometheus and Grafana instances are created."
   default     = true
 }
+
+variable "random_password_length" {
+  type        = number
+  description = "Length of the random generated password to use for app_secret_key"
+  default     = 16
```

Comment:
The default should be at least 32 or Play won't start once we merge in v2.9. I'd suggest making it 64 so we're set if we ever need to increate the signature algorithm from HS256 to HS512.

Reply:
I just don't want to break any existing deployments with the session stuff, but I think the plan is to change this default to higher once we know the implications and then do the play migration.